src/sketch.js

```diff
@@ -1,21 +1,28 @@
 /**
  * p5.js setup.
- * 
+ *
  * Returns:
  *  undefined.
  */
-function setup() 
-{
+let song;
```

Comment:
Please add a Constants and Definitions sections to this sketch.

```suggestion
/**
* CONSTANTS AND DEFINITIONS
*/
let song;
```